src/styles/card.css

```diff
@@ -0,0 +1,122 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 3px;
+
+  display: flex;
+  align-content: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.card__content {
+  display: flex;
+  align-content: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: stretch;
+
+  width: 170px;
```

Comment:
you shouldn't specify width for inner content, it should be 100%
you need to specify correct paddings for the main card, and this element (card__content wrapper) will be redundant